README.md

```diff
@@ -43,8 +43,11 @@ The local echo controller tries to replicate most of the bash-like user experien
     const term = new Terminal();
     term.open(document.getElementById('terminal'));
 
-    // Create a local echo controller
+    // Create a local echo controller (xterm.js v3)
     const localEcho = new LocalEchoController(term);
+    // Create a local echo controller (xterm.js >=v4)
+    const localEcho = new LocalEchoController();
```

Comment:
What do you think instead of breaking the API (in order to maintain some seamless migration from v3 to v4) to move this detection in the constructor?

For instance:
```
if (!term.onData) {
   // We are dealing with v4 API
} else {
   // We are dealing with v3 API
}
```

Reply:
I'd think that people using this library would be the ones specifying the version of xterm that they're using, and even if they aren't, passing the terminal to the constructor should still work even in v4.